services/AUTOMATIC1111/Dockerfile

```diff
@@ -41,7 +41,7 @@ RUN --mount=type=cache,target=/cache --mount=type=cache,target=/root/.cache/pip
 RUN --mount=type=cache,target=/root/.cache/pip \
   git clone https://github.com/AUTOMATIC1111/stable-diffusion-webui.git && \
   cd stable-diffusion-webui && \
-  git reset --hard 5ef669de080814067961f28357256e8fe27544f4 && \
+  git reset --hard bda2ecdbf58fd33b4ad3036ed5cc13eef02747ae && \
```

Comment:
latest one is `4afaaf8a020c1df457bcf7250cb1c7f609699fa7`, which furthermore includes updated changelog https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/4afaaf8a020c1df457bcf7250cb1c7f609699fa7, see https://github.com/AUTOMATIC1111/stable-diffusion-webui/commits/master

and I think you have to change it in line 73, too, no?

```
ARG SHA=5ef669de080814067961f28357256e8fe27544f4
->
ARG SHA=4afaaf8a020c1df457bcf7250cb1c7f609699fa7
```

Reply:
Thank you @jinnnyang and @cyril23, yes we need to change those, I will add a commit to this branch that addresses the issue.

Reply:
I had to do a lot of refactoring in #610 and will use that one instead of this, thank you again.